tremor-cli/tests/integration/window_empty_windows/config.yaml

```diff
@@ -0,0 +1,21 @@
+onramp:
```

Comment:
since those are purely testing pipelines, can we move that to a `test`? They run significantly faster, are easier to run, are easier to add and maintain, and easier to understand than cli-integration tests. They also add to code coverage :P

Reply:
Turns out we can't as the tests dont create any tick signals that we need here for those time based windows. :(